ocs_ci/ocs/resources/mcg.py

```diff
@@ -550,7 +550,7 @@ def _check_mirroring():
             return all(results)
 
         try:
-            for mirroring_is_complete in TimeoutSampler(140, 5, _check_mirroring):
+            for mirroring_is_complete in TimeoutSampler(140, 10, _check_mirroring):
```

Comment:
From what I observed the cmd wouldn't execute quickly enough sometimes (>5 secs) - thus extended the wait just for the sake of cleaner logs and easier tracking.